git_pull/Dockerfile

```diff
@@ -1,4 +1,9 @@
 ARG BUILD_FROM
+FROM $BUILD_FROM AS builder
+
+RUN apk add --no-cache go git musl-dev
+RUN go get github.com/adnanh/webhook
```

Comment:
Why not simply get the binary releases from GitHub?

Reply:
Good idea. I am struggeling to fetch the proper release depending on the architecture..
Or do I need to create a separate Dockerfile for each architecture then?

Reply:
Okay, I see we could download the release depending on the `$BUILD_ARCH` build arg.

But I am not fully convinced since the build process of the binary releases does not seem to be public, but the source code is.

Reply:
Agreed. Didn't realize it was a manual build process on their end. 👍